templates/workspace_services/guacamole/terraform/web_app.tf

```diff
@@ -34,6 +34,12 @@ resource "azurerm_app_service" "guacamole" {
     KEYVAULT_URL = local.kv_url
     API_URL      = local.api_url
     SERVICE_ID   = "${var.tre_resource_id}"
+    WORKSPACE_ID = "${var.workspace_id}"
+
+    # Webapp with pe won't pull the guacamole image without these
+    DOCKER_REGISTRY_SERVER_URL      = "${data.azurerm_container_registry.mgmt_acr.login_server}"
+    DOCKER_REGISTRY_SERVER_USERNAME = "${data.azurerm_container_registry.mgmt_acr.admin_username}"
+    DOCKER_REGISTRY_SERVER_PASSWORD = "${data.azurerm_container_registry.mgmt_acr.admin_password}"
```

Comment:
I am confused by the comment at top and the requirement to provide user/pass as we're not doing it elsewhere (like the API).

Reply:
You are right, probably it not the best practice and since I am leaving I opened a new bug about it (#861) and left it out of this PR.